README.md

```diff
@@ -957,8 +957,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map((number) => {
-      const nextNumber = number + 1;
-      `A string containing the ${nextNumber}.`;
```

Comment:
this change would get rid of the entire point of showing this bad example.

Reply:
no

without return this function is invalid

![grim--20-03-26--04-41-44](https://user-images.githubusercontent.com/1476565/77601567-293f4080-6f1c-11ea-8d69-12330a987aaf.png)

returning a single expression in the function body violates the rule

Reply:
That is the *whole point* - that it is an invalid example according to the guide (although it's perfectly valid javascript without the return). See the `// bad` above it?